AWSTranscribeStreaming/Internal/AWSSRWebSocketDelegateAdaptor.m

```diff
@@ -70,7 +74,7 @@ - (void)webSocket:(AWSSRWebSocket *)webSocket didReceiveMessage:(id)message {
 
  @param webSocket the web socket that opened
  */
-- (void)webSocketDidOpen:(AWSSRWebSocket *)webSocket {
+- (void)webSocketConnected:(AWSSRWebSocket *)webSocket {
```

Comment:
Why do we need this extra layer of indirection here? If we're adapting one protocol to another, it makes sense, but isn't this just taking an incoming event from SocketRocket and invoking a client delegate method?

Reply:
idk why I changed this. changing back. too many refactors heh sorry

Reply:
I disagree; the point of this class is to implement socketRocket delegate methods and forward them to the transcribe streaming delegate. By renaming, it actually makes the intent of the code *less* clear, especially since it's confusing what attempting to distinguish between "Connected" and "DidOpen".